docs/gettingstarted/query_test_report.rst

```diff
@@ -52,16 +52,16 @@ The **buildtest report --detailed** option can be used to show a detailed test r
 examining a test. This option is synonymous to running ``buildtest report --format name,id,user,state,returncode,runtime,outfile,errfile,buildspec``.
 Shown below is an example output.
 
-    .. dropdown:: ``buildtest report --detailed``
-        .. command-output:: buildtest report --detailed
+.. dropdown:: ``buildtest report --detailed``
+    .. command-output:: buildtest report --detailed
 
-    .. note::
-        The --detailed and --format options are mutually exclusive options because both options will alter the format
-        fields when displaying test results.
+.. note::
+        The ``--detailed`` and ``--format`` options are mutually exclusive options because both options
+        will alter the format fields when displaying test results.
 
 You will get the following error if you specify both options as shown below
-    .. dropdown::``buildtest report --detailed --format name``
 
+.. dropdown:: ``buildtest report --detailed --format name``
```

Comment:
you need a new line after this see https://buildtest--1567.org.readthedocs.build/en/1567/gettingstarted/query_test_report.html

It doesnt render this properly. Please fix this

![Screenshot 2023-07-31 at 2 01 09 PM](https://github.com/buildtesters/buildtest/assets/12942230/4c103858-a191-430c-92b3-8c554a0b7b62)